tests/test_bounds.py

```diff
@@ -37,6 +37,12 @@ def test_bounds_property_returns_map_of_coordinate_key_to_bounds_dataarray(self)
         for key in expected.keys():
             assert result[key].identical(expected[key])
 
+    def test_names_property_returns_a_list_of_bounds_names(self):
```

Comment:
Update func name to `def test_names_property_returns_a_list_of_sorted_bounds_names(self)`